src/BinaryParsers/ProgramDatabase/Pdb.cs

```diff
@@ -275,7 +275,18 @@ public bool IsSegmentWithIdWritable(uint addressSection)
         private HashSet<uint> GenerateExecutableSectionContribIds()
         {
             var result = new HashSet<uint>();
-            var enumSectionContribs = CreateDiaTable<IDiaEnumSectionContribs>();
+            IDiaEnumSectionContribs enumSectionContribs = null;
+
+            try
+            {
+                CreateDiaTable<IDiaEnumSectionContribs>();
```

Comment:
Danger! `enumSectionContribs` is always `null`; you probably meant to assign to it in the try block. (Did the compiler not warn about this?)